src/components/DeckMechanisms/index.js

```diff
@@ -141,6 +142,16 @@ export default class DeckMechanisms extends React.Component {
           newState.mana -= card.mana
         }
 
+        if (state.turn === 1) {
+          // Check if this card spawns units on the board, this is used to check if
+          // Toxic Sacrifice can be played on this turn
+          const unitSpawningSpells = ['N2', 'S24', 'F8']
```

Comment:
~I don’t think we should scope that solely on spells though. Playing Green Prototypes + Toxic Sacrifice is technically legitimate, so we really should check any card that can put a unit on the board and cost 2 mana or less. What do you think?~

I just saw the `card.type === 'unit' ||` below. Good thinking!